src/user_database.ts

```diff
@@ -493,3 +496,109 @@ export class KnexUserDatabase implements UserDatabase {
     return Promise.reject(new Error("dropSchema() is not supported in Knex user database."));
   }
 }
+
+export interface DrizzleClient {
```

Comment:
If we already include Drizzle as a dependency, do we still need to manually define an interface?

Reply:
Can we make it not a dependency?

Reply:
Drizzle is [tiny](https://bundlephobia.com/package/drizzle-orm@0.32.2) and has no dependencies, so I'm fine with depending on it